addons/website/static/src/js/menu/navbar.js

```diff
@@ -40,16 +40,14 @@ var WebsiteNavbar = publicWidget.RootWidget.extend({
      */
     start: function () {
         var self = this;
-        dom.initAutoMoreMenu(this.$('ul.o_menu_sections'), {
-            maxWidth: function () {
-                // The navbar contains different elements in community and
-                // enterprise, so we check for both of them here only
-                return self.$el.width()
-                    - (self.$('.o_menu_systray').outerWidth(true) || 0)
-                    - (self.$('ul#oe_applications').outerWidth(true) || 0)
-                    - (self.$('.o_menu_toggle').outerWidth(true) || 0)
-                    - (self.$('.o_menu_brand').outerWidth(true) || 0);
-            },
+        dom.initAutoMoreMenu(this.el.querySelector('ul.o_menu_sections'), {
+            // The navbar contains different elements in community and
+            // enterprise, so we check for both of them here only
+            maxWidth: self.$el.width()
+                - (self.$('.o_menu_systray').outerWidth(true) || 0)
+                - (self.$('ul#oe_applications').outerWidth(true) || 0)
+                - (self.$('.o_menu_toggle').outerWidth(true) || 0)
+                - (self.$('.o_menu_brand').outerWidth(true) || 0),
```

Comment:
Ok, this one I am not sure it entirely is useful since there are not many items in the purple navbar by default but it could with customization. It does not work anymore.

1. The maxWidth with a function may be useful? When I add fake menu item via my console, it simply does not do anything anymore
2. When I add item in the views definition and reload the page, it works... but then it crashes (traceback) :grimacing:

Reply:
1- Yes you're right ! :+1: 

2- The traceback is caused by some items in user navbar that are not "dropdown-item".
Actually the use of jQuery in old code hides this kind of issues... I added some code to handle properly items in user navbar and make them visible when added to the "+" button.